.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12, 14]
+    name: Node ${{ matrix.node }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use node ${{ matrix.node }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install
+        run: npm install
+      - name: Test
+        if: ${{ matrix.node != 14}}
+        run: npm test
+      - name: Test Browsers
+        if: ${{ matrix.node == 14}}
+        env:
+          SAUCE_CONNECT_DOWNLOAD_ON_INSTALL: true
```

Comment:
I prefer a separate workflow for the browser tests (making it easier to retry just those tests). Also needs secrets (I will create them) and adding a hostname to `hosts`.

To achieve that we can copy [`nodejs/readable-stream/.github/workflows/sauce.yml`](https://github.com/nodejs/readable-stream/blob/master/.github/workflows/sauce.yml) as-is 😃

Reply:
I've brought that workflow over into the PR. I think it's better for them to be separate as well.